apps/generic-issuance-client/build.ts

```diff
@@ -0,0 +1,61 @@
+import { NodeGlobalsPolyfillPlugin } from "@esbuild-plugins/node-globals-polyfill";
+import { NodeModulesPolyfillPlugin } from "@esbuild-plugins/node-modules-polyfill";
+import { build, BuildOptions, context } from "esbuild";
+import fs from "fs";
+
+const consumerClientAppOpts: BuildOptions = {
```

Comment:
This says "consumerClient" but should probably say "genericIssuanceClient"

Reply:
good catch - updated!